src/systems/physics/Physics.cc

```diff
@@ -1365,8 +1390,12 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
         auto canonicalPoseComp =
             _ecm.Component<components::Pose>(linkEntityIt->second);
```

Comment:
From a quick check this variable seems unused now, can it be removed?

Reply:
removed. 7740d5cc86ab37ed863e13e377bb551367c2367b